src/bucket.ts

```diff
@@ -264,6 +264,11 @@ export enum BucketActionToHTTPMethod {
   list = 'GET',
 }
 
+export enum Methods {
```

Comment:
The name `Methods` seems vague/confusing to me if I was picking this up knowing nothing, but I'm not sure what a better option would be.